File: Ser2023/Company/Company.Web/Modules/Store/CategoriesType/CategoriesTypeColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Company.Store.Columns;

[ColumnsScript("Store.CategoriesType")]
[BasedOnRow(typeof(CategoriesTypeRow), CheckNames = true)]
public class CategoriesTypeColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public Int32 CategoryTypeId { get; set; }
    public Int16 Type { get; set; }
    [EditLink]
    public String CategoryType { get; set; }
}

File: 2026/Company/Company.Web/Modules/ServerTypes/Sales/CategoriesTypeColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { CategoriesTypeRow } from "./CategoriesTypeRow";

export interface CategoriesTypeColumns {
    CategoryTypeId: Column<CategoriesTypeRow>;
    Type: Column<CategoriesTypeRow>;
    CategoryType: Column<CategoriesTypeRow>;
}

export class CategoriesTypeColumns extends ColumnsBase<CategoriesTypeRow> {
    static readonly columnsKey = 'Sales.CategoriesType';
    static readonly Fields = fieldsProxy<CategoriesTypeColumns>();
}